packages/guides-code/resources/template/html/body/code.html.twig

```diff
@@ -0,0 +1,13 @@
+{% if node.raw %}
+    {# see the RawDirective for where this is coming from; a refactor is desired to move this onto its own template / renderer #}
+    {{ node.value|raw }}
+{%- else -%}
+    {% if node.caption %}
+        <div class="code-block-caption">
+            <span class="caption-text">{{ node.caption }}</span>
+        </div>
+    {%- endif -%}
+    <pre{% if node.classes %} class="{{ node.classesString }}"{% endif %}><code class="language-{{ node.language }}{{ node.startingLineNumber ? ' line-numbers' }}"
+                {%- if node.startingLineNumber %} data-start="{{ node.startingLineNumber }}"{% endif -%}
+                {%- if node.emphasizeLines %} data-emphasize-lines="{{ node.emphasizeLines }}"{% endif -%}>{{ node.value|highlight(node.language) }}</code></pre>
```

Comment:
The change is just on this line, maybe we should introduce a template block and allow this extension to extend the base template?

Reply:
that is a good idea, applied it